hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java

```diff
@@ -7963,7 +7963,9 @@ public Result append(Append mutate, long nonceGroup, long nonce) throws IOExcept
         for (Map.Entry<Store, List<Cell>> entry: removedCellsForMemStore.entrySet()) {
           entry.getKey().add(entry.getValue());
         }
-        if (we != null) mvcc.complete(we);
+        if (we != null) {
```

Comment:
Changes to this file are a carrying forward of formatting changes from the branch-1.3 changes proposed on HBASE-22508. They don't address a findbugs issue, only try to keep the code as consistent as possible between branches.